steps/reasons-for-appealing/evidence-upload/EvidenceUpload.js

```diff
@@ -0,0 +1,122 @@
+const { Question, goTo } = require('@hmcts/one-per-page');
+const { form, text } = require('@hmcts/one-per-page/forms');
+const { Logger } = require('@hmcts/nodejs-logging');
+const { api } = require('config/default.json');
```

Comment:
Config does not work like this, there's no need to explicitly define `default.json` or `custom-environment-variables.json` or any other environment as part of the require. Config does this for you based on the NODE_ENV environment variable.

Locally
```
const config = require('config');
const url = config.get('api.url'); // http://localhost:8080
const uploadEvidenceUrl = config.get('api.uploadEvidenceUrl'); // http://localhost:3010/upload
```
NODE_ENV=aat or prod 
```
const config = require('config');
const url = config.get('api.url'); // Comes from the env. var. TRIBUNALS_CASE_API_URL 
const uploadEvidenceUrl = config.get('api.uploadEvidenceUrl'); // This should come from an environment variable (you haven't created this)
```